server/controllers/Messages/inputs.ts

```diff
@@ -0,0 +1,15 @@
+import { Length } from 'class-validator';
+import { InputType, Field } from 'type-graphql';
+
+@InputType()
+export class SendEmailInputs {
+  @Field(() => [String])
+  to: String;
```

Comment:
```suggestion
  to: [string];
```
TypeScript was complaining with just `String`